pkg/sdk/testint/tasks_gen_integration_test.go

```diff
@@ -338,120 +344,180 @@ func TestInt_Tasks(t *testing.T) {
 	})
 
 	t.Run("clone task: default", func(t *testing.T) {
-		sourceTask := createTask(t)
+		sourceTask, taskCleanup := testClientHelper().Task.Create(t)
+		t.Cleanup(taskCleanup)
 
 		id := testClientHelper().Ids.RandomSchemaObjectIdentifier()
-
-		request := sdk.NewCloneTaskRequest(id, sourceTask.ID())
-
-		err := client.Tasks.Clone(ctx, request)
+		err := client.Tasks.Clone(ctx, sdk.NewCloneTaskRequest(id, sourceTask.ID()))
 		require.NoError(t, err)
-		t.Cleanup(cleanupTaskProvider(id))
+		t.Cleanup(testClientHelper().Task.DropTaskFunc(t, id))
 
 		task, err := client.Tasks.ShowByID(ctx, id)
 		require.NoError(t, err)
 
-		assertTask(t, task, request.GetName())
+		assertTask(t, task, id, testClientHelper().Ids.WarehouseId().Name())
 	})
 
-	t.Run("drop task: existing", func(t *testing.T) {
-		request := createTaskBasicRequest(t)
-		id := request.GetName()
-
-		err := client.Tasks.Create(ctx, request)
+	t.Run("create or alter: complete", func(t *testing.T) {
+		id := testClientHelper().Ids.RandomSchemaObjectIdentifier()
+		err := client.Tasks.CreateOrAlter(ctx, sdk.NewCreateOrAlterTaskRequest(id, sql).
+			WithWarehouse(*sdk.NewCreateTaskWarehouseRequest().WithWarehouse(testClientHelper().Ids.WarehouseId())).
+			WithSchedule("10 MINUTES").
+			WithConfig(`$${"output_dir": "/temp/test_directory/", "learning_rate": 0.1}$$`).
+			WithAllowOverlappingExecution(true).
```

Comment:
I've now noticed that ERROR_INTEGRATION is missing from multiple places.

Reply:
True, added integration to some of the tests and changed SDK a bit to make it clear it's notification integration (there's no such thing as error integration).

Reply:
OK. In the docs we have ERROR_INTEGRATION, so maybe it's worth reporting on docs discuss channel.